lib/auth/strategies/api-user-jwt.js

```diff
@@ -20,39 +20,33 @@ module.exports = (srv, options) => {
              */
             validateFunc: (decoded, request, reply) => {
 
-                const Model = request.collections();
-                const Tokens = Model.tokens;
-                const Users = Model.users;
+                const Tokens = request.models().Tokens;
```

Comment:
Subtle shit, but we (CC @WilliamSWoodruff) found that this needs to be `request.models(true).Tokens` because the `request` might not come from a route in this plugin.  The other option would be to use `srv.models().Tokens`, which is not request-dependent.